cmd/utils/config_test.go

```diff
@@ -0,0 +1,272 @@
+		"cypress":                                   true,
+		"baobab":                                    true,
+		"block-generation-interval":                 true,
+		"block-generation-time-limit":               true,
+		"txresend.interval":                         true,
+		"txresend.max-count":                        true,
+		"txresend.use-legacy":                       true,
+		"txpool.spamthrottler.disable":              true,
+		"scsigner":                                  false,
+		"childchainindexing":                        true,
+		"mainbridge":                                true,
+		"mainbridgeport":                            true,
+		"kes.nodetype.service":                      true,
+		"dbsyncer":                                  true,
+		"dbsyncer.db.host":                          true,
+		"dbsyncer.db.port":                          true,
+		"dbsyncer.db.name":                          true,
+		"dbsyncer.db.user":                          true,
+		"dbsyncer.db.password":                      true,
+		"dbsyncer.logmode":                          true,
+		"dbsyncer.db.max.idle":                      true,
+		"dbsyncer.db.max.open":                      true,
+		"dbsyncer.db.max.lifetime":                  true,
+		"dbsyncer.block.channel.size":               true,
+		"dbsyncer.mode":                             true,
+		"dbsyncer.genquery.th":                      true,
+		"dbsyncer.insert.th":                        true,
+		"dbsyncer.bulk.size":                        true,
+		"dbsyncer.event.mode":                       true,
+		"dbsyncer.max.block.diff":                   true,
+		"chaintxperiod":                             true,
+		"chaintxlimit":                              true,
+		"subbridge":                                 true,
+		"subbridgeport":                             true,
+		"parentchainid":                             true,
+		"vtrecovery":                                true,
+		"vtrecoveryinterval":                        true,
+		"scnewaccount":                              true,
+		"anchoring":                                 true,
+		"sc.parentoperator.gaslimit":                true,
+		"sc.childoperator.gaslimit":                 true,
+		"kas.sc.anchor":                             false,
+		"kas.sc.anchor.period":                      false,
+		"kas.sc.anchor.url":                         false,
+		"kas.sc.anchor.operator":                    false,
+		"kas.secretkey":                             false,
+		"kas.accesskey":                             false,
+		"kas.x-chain-id":                            false,
+		"kas.sc.anchor.request.timeout":             false,
+	}
+
+	printFlags := func(ctx *cli.Context) error {
+		for _, flag := range AllNodeFlags() {
+			expected := testcases[flag.Names()[0]]
+			actual := ctx.IsSet(flag.Names()[0])
+			if expected != actual {
+				return fmt.Errorf("flag IsSet expected %t, but return %t", expected, actual)
+			}
```

Comment:
What about using assert.Equal()?
```suggestion
			assert.Equal(t, testcases[flag.Names()[0]], ctx.IsSet(flag.Names()[0]), "IsSet returned unexpected result")
```

Reply:
5afbed2c5c92c74839b9b4df1027bf423f339a3d